Emphasize in docs that in clustering environments, when using incremental import the updated database must be reseeded (#2093)
neo4j/docs-operations: modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

=== Usage and limitations

[WARNING]
====
The importer works well on standalone servers.

> In clustering environments with multiple copies of the database, the updated database *must be reseeded*.

Reviewer: This is my wild guess, but I think we have to clarify that the database must be reseeded from a specific backup.

Author: I think you reseed the other databases from the one you have updated.

Reviewer: So, the updated database must be used as a source for reseed?

Author: Yes, that's how I understand it. But @tinwelint can chime in here.

Reviewer 2: Yes, the database you imported new data into is the only database that have the new data, right? So that's the one that has to be the seed source.

> In clustering environments with multiple copies of the database, the updated database *must be reseeded*.

Author: ```suggestion
In clustering environments with multiple copies of the database, the updated database must be used as a source to reseed the rest of the database copies.
```

> You can use the procedure xref:procedures.adoc#procedure_dbms_cluster_recreateDatabase[dbms.cluster.recreateDatabase()] to reseed the database.

Author: ```suggestion
You can use the procedure xref:procedures.adoc#procedure_dbms_cluster_recreateDatabase[dbms.cluster.recreateDatabase()] to reseed the databases.
```

> You can use the procedure xref:procedures.adoc#procedure_dbms_cluster_recreateDatabase[dbms.cluster.recreateDatabase()].

Reviewer: ```suggestion
You can use the procedure xref:procedures.adoc#procedure_dbms_cluster_recreateDatabase[`dbms.cluster.recreateDatabase()`].
```

> You can use the procedure xref:procedures.adoc#procedure_dbms_cluster_recreateDatabase[dbms.cluster.recreateDatabase()] to reseed the database copies.

Author: ```suggestion
You can use the procedure xref:procedures.adoc#procedure_dbms_cluster_recreateDatabase[dbms.cluster.recreateDatabase()].
```

For details, see xref:clustering/databases.adoc#recreate-databases[Recreate databases].

> Starting the clustered database after an incremental import, without reseeding, or performing the incremental import on a single server while the database remains online on other clustered members, may result in unpredictable consequences, which may include data inconsistency between cluster members.

Reviewer: ```suggestion
Starting the clustered database after an incremental import without reseeding or performing the incremental import on a single server while the database remains online on other clustered members may result in unpredictable consequences, including data inconsistency between cluster members.
```
